### DISCUSSION TITLE
Welcome to the STM32 MCU Discussions!

### DISCUSSION THREAD

**Discussion by ALABSTM** [STMicroelectronics/STM32Cube_MCU_Overall_Offer#16] [Announcements]
## :wave: Welcome!

We're using "Discussions" as a way to **make announcements** and to **share messages with our community** about the STM32Cube MCU software published within this organization.

We hope that you find it useful and interesting. Please let us know your feedback.

Enjoy our **STM32Cube** offer and stay tuned!
##
**PS:** Best wishes for the new year!

**Comment by Alex1881** [comment]
Why should I be able to make comments on your announcements as a user??
And where is the place to have technical discussions? 

PS: Repurposing features of a platform for workarounds is a bit fishy why don't you guys build a real page on GitHub, with a real structure, announcements, informations, guidance for the users ?
